fury-on-gke/README.md

```diff
@@ -379,88 +386,82 @@ furyctl vendor -H
 2. Inspect the downloaded modules in the `vendor` folder:
 
 ```bash
-tree -d /demo/vendor -L 3
+tree -d /demo/vendor -L 2
 ```
 
 Output:
 
 ```bash
-$ tree -d vendor -L 3
+$ tree -d vendor -L 2
 
 vendor
 └── katalog
     ├── ingress
-    │   ├── cert-manager
-    │   ├── forecastle
-    │   └── nginx
     ├── logging
-    │   ├── cerebro
-    │   ├── curator
-    │   ├── elasticsearch-single
-    │   ├── fluentd
-    │   └── kibana
     ├── monitoring
-    │   ├── alertmanager-operated
-    │   ├── configs
-    │   ├── goldpinger
-    │   ├── grafana
-    │   ├── kube-proxy-metrics
-    │   ├── kube-state-metrics
-    │   ├── node-exporter
-    │   ├── prometheus-operated
-    │   └── prometheus-operator
-    └── networking
-        └── calico
+    ├── networking
+    └── opa
+
 ```
 
 ## Step 3 - Installation
 
-Each module is a Kustomize project. Kustomize allows to group together related Kubernetes resources and combine them to create more complex deployment. Moreover, it is flexible, and it enables a simple patching mechanism for additional customization.
+Each module is a Kustomize project. Kustomize allows grouping together related Kubernetes resources and combining them to create more complex deployments. Moreover, it is flexible, and it enables a simple patching mechanism for additional customization.
 
 To deploy the Fury distribution, use the main `manifests/demo-fury/kustomization.yaml` file:
 
 ```yaml
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
 resources:
 
-# Ingress module
+# Ingress
 - ../vendor/katalog/ingress/forecastle
 - ../vendor/katalog/ingress/nginx
 - ../vendor/katalog/ingress/cert-manager
 
-# Logging module
+# Logging
+- ../vendor/katalog/logging/opensearch-single
+- ../vendor/katalog/logging/opensearch-dashboards
+- ../vendor/katalog/logging/logging-operator
+- ../vendor/katalog/logging/logging-operated
+- ../vendor/katalog/logging/configs
 - ../vendor/katalog/logging/cerebro
-- ../vendor/katalog/logging/curator
-- ../vendor/katalog/logging/elasticsearch-single
-- ../vendor/katalog/logging/fluentd
-- ../vendor/katalog/logging/kibana
 
-# Monitoring module
-- ../vendor/katalog/monitoring/alertmanager-operated
-- ../vendor/katalog/monitoring/goldpinger
+# Monitoring
+- ../vendor/katalog/monitoring/prometheus-operator
+- ../vendor/katalog/monitoring/prometheus-operated
 - ../vendor/katalog/monitoring/grafana
-- ../vendor/katalog/monitoring/kube-proxy-metrics
 - ../vendor/katalog/monitoring/kube-state-metrics
 - ../vendor/katalog/monitoring/node-exporter
-- ../vendor/katalog/monitoring/prometheus-operated
-- ../vendor/katalog/monitoring/prometheus-operator
+- ../vendor/katalog/monitoring/alertmanager-operated
+- ../vendor/katalog/monitoring/kube-proxy-metrics
+- ../vendor/katalog/monitoring/eks-sm
```

Comment:
I think we need to use the gke-sm one here

Reply:
:facepalm: thanks!